modules/farm_fd2_school/src/entrypoints/HTML_CSS/App.vue

```diff
@@ -1,10 +1,65 @@
 <template>
-  <div
-    id="HTML-CSS"
-    data-cy="HTML-CSS"
-  >
-    <p>Add HTML for Harvest form here.</p>
-  </div>
+  <head>
+    <title>Harvest</title>
+  </head>
+  <p>
+    <label for="start">Date:</label>
+
+    <input
+      type="date"
+      id="start"
+      name="crop-start"
+      value="2019-06-15"
+      min="2019-01-01"
+      max="2019-12-31"
+    />
+  </p>
+  <p>
+    <label for="crop-select">Crop:</label>
+
+    <select
+      name="crops"
+      id="Crop"
+    >
+      <option value="radish">RADISH</option>
```

Comment:
It would be good to use the `selected` attribute here rather than reordering the options in the list so that RADISH is first.